src/v/archival/upload_housekeeping_service.cc

```diff
@@ -167,13 +173,15 @@ void upload_housekeeping_service::epoch_timer_callback() {
 void upload_housekeeping_service::register_jobs(
   std::vector<std::reference_wrapper<housekeeping_job>> jobs) {
     for (auto ref : jobs) {
+        vlog(_ctxlog.info, "Registering job: {}", ref.get().name());
```

Comment:
INFO is kind of high for an >= per-NTP log about internals, although it's not too bad if it's just at startup/shutdown.  But will we also get these Registering... logs on e.g. leadership changes?  That could be a bit too chatty.